hexrd/config/findorientations.py

```diff
@@ -235,3 +235,10 @@ def file(self):
     @property
     def threshold(self):
         return self._cfg.get('find_orientations:orientation_maps:threshold')
+
+    @property
+    def filter_maps(self):
+        return self._cfg.get('find_orientations:orientation_maps:filter_maps',
+                             default=False)
```

Comment:
We should add this as an option in hexdgui I am guessing?

Reply:
yeah, we should update the widget.  In the filtering there are two options:

1. row-wise median filtering to get rid of the streaking artifacts; and
2. the above plus filtering with `gauss_laplace` with a user-specified stdev